src/ensembl/src/content/app/entity-viewer/state/sidebar/entityViewerSidebarActions.ts

```diff
@@ -114,3 +114,51 @@ export const updateEntityUI: ActionCreator<ThunkAction<
 
   dispatch(updateEntityUIState({ genomeId, entityId, fragment }));
 };
+
+export const openSidebarModal: ActionCreator<ThunkAction<
```

Comment:
🙅‍♂️ ActionCreator

```
export const openSidebarModal = (sidebarModalView: string): ThunkAction<
  void,
  any,
  null,
  Action<string>
>  => (dispatch, getState: () => RootState) => ...
```

Reply:
Ok is this how it should be used? Shall I change every actions in that file? @azangru

Reply:
changed all actions now